brave/brave-core
# Monthly statement

Resolves https://github.com/brave/brave-browser/issues/930

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- do AC, recurring tip and one time tip
- click on monthly report and make sure that all data is displayed correctly 

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
